main.cpp

```diff
@@ -56,10 +54,27 @@ void variableDeclarations()
 {
     //example:
     int number = 2; //declaration of a primitive named 'number' with an initial value of '2'
+    int startDay = 24;
+    int age = 12;
+    int shoes = 0;
+
+    char note = 64;
+    char velocity = 127;
+    char channel = 16;
```

Comment:
`char` is a single alpha-numeric symbol surrounded by single quotes: `'a'`
- [ ] revise